src/Compilers/Core/Portable/Symbols/IFunctionPointerTypeSymbol.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.CodeAnalysis
 {
     // PROTOTYPE(func-ptr): Document
     // https://github.com/dotnet/roslyn/issues/39865: Expose calling convention on either this or IMethodSymbol in general
-    public interface IFunctionPointerTypeSymbol : ISymbol
+    public interface IFunctionPointerTypeSymbol : ITypeSymbol
```

Comment:
❔ Should this be `INamedTypeSymbol` to match delegates?

Reply:
I wouldn't think so, since they do behave differently (they don't have names to start!) @333fred?

Reply:
Absolutely not. They are not named types.